### DISCUSSION TITLE
.WithAppOnly Doesn't Exist

### DISCUSSION THREAD

**Discussion by mattbanes** [AzureAD/microsoft-identity-web#1330] [Q&A] [answered]
I'm trying to make a call using the Microsoft Graph SDK using application permissions instead of delegated. According to the information for version 1.2.0 I should be able to add .WithAppOnly after Request(), but I get an error message because .WithAppOnly doesn't exist. I'm guessing there is something simple I'm missing, but I haven't been able to find anything beyond the link below on this topic.

![image](https://user-images.githubusercontent.com/45860453/126535514-d6f28e69-dafe-49ff-b6ab-a91e74250c3c.png)

https://github.com/AzureAD/microsoft-identity-web/wiki/1.2.0#you-can-now-specify-scopes-and-app-permissions-for-graphserviceclient

**Comment by jmprieur** [comment, marked as answer]
@mattbanes : did you reference Microsoft.Identity.Web.MicrosoftGraph or Microsoft.Identity.Web.MicrosoftGraphBeta to your project?

**Reply by mattbanes** [reply]
@jmprieur I feel like a moron. I realized I was looking at the wrong project's Nuget packages when I thought I had that package added. That's why I wasn't getting any intellisense help on this. It's working as described now that the user error is resolved. 🙄
